### PR TITLE
ECC-1275| hint text for ROW org type

### PR DESCRIPTION
**Pull request by oliwiaKot**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -524,7 +524,7 @@ cds.matching.organisation-type.user.header=What do you want to apply as?
 cds.matching.organisation-type.organisation-type=Business type
 cds.matching.organisation-type.radio.company.label=Company
 cds.matching.organisation-type.radio.organisation.label=Organisation
-cds.matching.organisation-type.radio.organisation.hint-text=For example a limited company, corporation, limited liability company or charity
+cds.matching.organisation-type.radio.organisation.hint-text=For example, a limited company, corporation, limited liability company or charity
```
Comment (beckfordp) [inline conf/messages.en:527]: Are you sure we should add a comma here? Worth checking with Colin...Also the Welsh of course.

**Review by beckfordp — APPROVED** [review]
LGTM as long as the content is correct. May want to check.

File: conf/messages.en (hunk 1 above)
Comment (oliwiaKot) [reply conf/messages.en:527]: I checked with Colin, the comma should be there. The welsh is correct

Comment (beckfordp) [reply conf/messages.en:527]: 👍

**Pull request merged**